src/transformers/image_transforms.py

```diff
@@ -539,6 +539,51 @@ def _center_to_corners_format_tf(bboxes_center: "tf.Tensor") -> "tf.Tensor":
     return bboxes_corners
 
 
+# inspired by https://github.com/ViTAE-Transformer/ViTPose/blob/d5216452796c90c6bc29f5c5ec0bdba94366768a/mmpose/datasets/datasets/base/kpt_2d_sview_rgb_img_top_down_dataset.py#L132
+def box_to_center_and_scale(
+    box: Union[Tuple, List, np.ndarray],
+    image_width: int,
+    image_height: int,
+    pixel_std: float = 200.0,
```

Comment:
Not sure this should be in image_transformers, since a pixel std of 200 is a bit random

(didn't find the reason yet why they use a value of 200)

Reply:
@NielsRogge I also agree I think we can put this code in vitpose processor part?